code/modules/mining/ore.dm

```diff
@@ -16,6 +16,31 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(AM as mob|obj)
+	var/obj/item/weapon/storage/bag/ore/OB
+	var/turf/simulated/floor/F = get_turf(src)
+	if(loc != F)
+		return ..()
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
```

Comment:
except it doesn't return true for xenomorphs, which are definately "humanoid"
it literally just checks if it's a type of /human/ despite what ponies always insisted on saying